ci/travis/before_deploy.sh

```diff
@@ -39,7 +39,7 @@ if [ "${TRAVIS_BRANCH}" = "master" ]; then
 
     # Build all NuPIC and all required python packages into dist/wheels as .whl
     # files.
-    pip wheel --wheel-dir=dist/wheels -r requirements.txt
+    pip wheel --wheel-dir=dist/wheels -r requirements.txt #TODO do we need to run this file? 
```

Comment:
@dkeeney can you have a look, please, if we still need this file? Or does setup.py now do all of the packaging work?

Reply:
setup.py builds an egg not a wheel.
I am looking at maybe switching that to create a wheel since pip seems to prefer wheels.
So to answer your question..I don't know what I will need for setup.py to build wheels yet.